content/docs/2.15/reference/_index.md

```diff
@@ -6,8 +6,8 @@ weight = 2
 Reference information for the KEDA autoscaler.
 
 - [Scaled Object specification](./scaledobject-spec)
-- [ScaledJob specification](../concepts/scaling-jobs/#scaledjob-spec)
+- [ScaledJob specification](./scaledjob-spec.md)
```

Comment:
I think this is correct, but it should be double checked

Reply:
Yup, that's correct.